README.md

```diff
@@ -46,3 +46,26 @@ wanting to implement a Platform or Service Broker.
 The [Platform Compatibility](compatibility.md) document is a quick way to track
 the state of current and pending feature additions, how they align with
 releases, and which platforms are currently supporting them.
+
+## Testing conformance to the Open Service Broker API
+
+One of the [goals](https://www.openservicebrokerapi.org/#goals) of the Open
+Service Broker API project is to **provide conformance test suites to verify
+compliance with the specification**.
+
+The conformance test suites below can be used to verify Service Broker behaviour
+against different versions of the specification:
```

Comment:
Add a comment about PR'ing new ones in?

Reply:
Maybe depends on the ch checkers project specific contribution guidelines? So should be done on each projects start page? Maybe add this convention here?

Reply:
I'm not sure I understand @chbrinker - do you mean which versions?

Reply:
Was a misunderstanding. Did not read your question correctly. I was thinking you meant contributions to the checker frameworks. But now I think you meant contributing new frameworks to the list of checkers. Makes more sense. You should add this info.